draft-ietf-quic-multipath.md

```diff
@@ -616,7 +595,31 @@ be controlled by the combination of one or several of the following:
    uses a series of consecutive sequence numbers without creating
    holes.
 
-### ACK Delay Considerations
+### Loss and Congestion Handling With Zero-Length CID {#zero-length-cid-loss-and-congestion}
+
+When sending to a zero-length CID
+receiver, senders will receive back acknowledgements that combine packet
+numbers received over multiple paths. Senders MUST be able to infer the
+sending path from the acknowledged packet numbers, for example by remembering
+which packet was sent on what path. The senders MUST use that information to
+perform congestion control on the relevant paths, and to correctly
+estimate the transmission delays on each path. (See
+{{ack-delay-and-zero-length-cid-considerations}} for specific considerations
+about using the ACK Delay field of ACK frames, and
+{{ecn-and-zero-length-cid-considerations}} for issues on using ECN marks.)
+
+Loss detection as specified in {{QUIC-RECOVERY}} uses algorithms
+based on timers and on sequence numbers. When sending to zero-length CID receivers,
+senders expect that packets sent on different paths will not be received in
+order. They cannot directly use the packet sequence numbers to
+compute the Packet Thresholds defined in {{Section 6.1.1 of QUIC-RECOVERY}}.
+Relying only on Time Thresholds produces correct results, but is somewhat
+suboptimal. Some implementations have been getting good results by
+remembering not just the path over which a packet was sent, but also
+the order of a packet on that path. They can then use this order in the
+Packet Threshold tests.
```

Comment:
I don't think I fully understand these last couple of sentences. Can you clarify?

Reply:
Fixed.